sample/src/main/kotlin/com/slack/circuit/sample/petlist/PetList.kt

```diff
@@ -96,11 +96,25 @@ data class PetListAnimal(
   val imageUrl: String?,
   val breed: String?,
   val gender: String,
+  val size: String,
   val age: String,
 ) : Parcelable
 
+enum class Gender {
+  ALL,
+  MALE,
+  FEMALE
+}
+
+enum class Size {
+  ALL,
+  SMALL,
+  MEDIUM,
+  LARGE
+}
+
 @Parcelize
-object PetListScreen : Screen {
+data class PetListScreen(val gender: Gender = Gender.ALL, val size: Size = Size.ALL) : Screen {
```

Comment:
nit: can we wrap up filter options in a `Filters` data class for readability?

```kotlin
@Parcelize
class Filters(val gender: Gender = Gender.ALL, val size: Size = Size.ALL) : Parcelable
```

Makes it easier to change them later too

Reply:
Addressed here, https://github.com/slackhq/circuit/pull/101/commits/90a0d9229d82aea4234c9c04e58e2cdef2600978.